moveit_core/planning_request_adapter/include/moveit/planning_request_adapter/planning_request_adapter.h

```diff
@@ -52,7 +52,7 @@ class PlanningRequestAdapter
 {
 public:
   using PlannerFn =
-      std::function<bool(const planning_scene::PlanningSceneConstPtr&, const planning_interface::MotionPlanRequest&,
+      std::function<void(const planning_scene::PlanningSceneConstPtr&, const planning_interface::MotionPlanRequest&,
```

Comment:
Success/failure will be reported in the MotionPlanResponse now so it's not ambiguous